src/api/app/models/user.rb

```diff
@@ -901,8 +901,7 @@ def update_login_values(env)
   end
 
   def send_metric_for_beta_change
-    channel = (in_beta? ? 'joined_beta' : 'left_beta')
-    RabbitmqBus.send_to_bus('metrics', "user.#{channel} value=1")
+    RabbitmqBus.send_to_bus('metrics', "user in_beta=#{User.in_beta.count},count=#{User.count}")
```

Comment:
Your `in_beta` tag will have a very high cardinality if the number that `User.in_beta.count` returns is changing often.

A tag key should have not have many possible tag values.

Reply:
As far as I understood, the syntax to add tags would be just after the metric:

```
metric_name,tag1=value_tag1,tag2=value_tag2 field1=value_field1,field2=value_field2
```

The idea was to store the value in a field, and not in a tag. So we don't have tags in this case, but two fields: `in_beta` and `count`. Is that correct?

Reply:
https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/

Reply:
'metrics' is not the metric name. It's not part of what you send to influxdb. It's the bus you publish this on.

Reply:
Using real numbers, the line protocol could look like:
`user in_beta=5,count=100`

Where the metric name is `user`, there are no tags and the fields are `in_beta` and `count`. This way we don't have a problems with cardinality. But maybe you consider there is a better approach for this.

Reply:
Ah yes, sorry I was wrong. :man_facepalming: 

Then you just have the same "problems" as in #10353

Reply:
We fixed this, performing the measurements every 5 minutes. Could you have a look again?